lib/js_of_ocaml/typed_array.mli

```diff
@@ -254,3 +254,11 @@ module String : sig
 
   val of_uint8Array : uint8Array Js.t -> string
 end
+
+module Bytes : sig
```

Comment:
Maybe add documentations explaining that theses functions do not copy their inputs (most of the time) and that modifying the input/output might modify the corresponding output/input

Reply:
I just added a short paragraph that explains the behavior with a warning.